bin/oref0-pump-loop.sh

```diff
@@ -339,7 +339,11 @@ function refresh_old_profile {
 
 function refresh_smb_temp_and_enact {
     # set mtime of monitor/glucose.json to the time of its most recent glucose value
-    touch -d "$(date -R -d @$(jq .[0].date/1000 monitor/glucose.json))" monitor/glucose.json
+    if grep "MDT cgm" openaps.ini; then
+      touch -d "$(date -R -d @$(jq .[0].date/1000 nightscout/glucose.json))" monitor/glucose.json
```

Comment:
I am not concerned about memory usage, but rather code duplication. So I'd be fine with it being a sting, as long as we can reduce the number of places we do the same if check (and that will need to be updated together later if we change that).